geode-core/src/main/java/org/apache/geode/distributed/ConfigurationProperties.java

```diff
@@ -1943,6 +1931,36 @@
    * <U>Default</U>: ""
    */
   String REDIS_PASSWORD = "redis-password";
+  /**
+   * The static String definition of the <i>"redis-port"</i> property <a name="redis-port"/a>
+   * </p>
+   * <U>Description</U>: Specifies the port used by Redis API for Geode which enables Redis
+   * clients to connect and store data in GemFire distributed system. A value of 0 will select a
+   * random port.
+   * </p>
+   * <U>Default</U>: "6379"
+   * </p>
+   * <U>Allowed values</U>: 0..65535
+   */
+  String REDIS_PORT = "redis-port";
+  /**
+   * The static String definition of the <i>"redis-service-enabled"</i> property <a
+   * name="redis-service-enabled"/a>
+   * </p>
+   * <U>Description</U>: When the default value of false, the Redis API for Geode is not available.
+   * Set to true to enable the Redis API for Geode.</td>
+   * </p>
+   * <U>Default</U>: false
+   * <td>redis-service-enabled</td>
+   * <td>When the default value of false, the Redis API for <%=vars.product_name%> is not available.
+   * Set
+   * to true to enable the Redis API for <%=vars.product_name%>.</td>
+   * <td>S</td>
+   * <td>false</td>
+   * </tr>
+   * </p>
+   */
+  String REDIS_SERVICE_ENABLED = "redis-service-enabled";
```

Comment:
Why add in the word "service" here? Why not just "redis-enabled". None of the other redis properties use the word "service". I think consistent naming conventions are really important to avoid confusion.

Reply:
Yeah, we can make it `redis-enabled`